<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisForm</class>
 <widget class="QWidget" name="analysisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="formTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Панель анализа</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Анализ</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelDistanceBetweenCenterMass">
        <property name="text">
         <string>Расстояние между центрами масс контуров </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLifeTime">
        <property name="text">
         <string>Время жизни </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxLifeTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelMinSizeCrownCharge">
        <property name="text">
         <string>Минимальный размер заряда</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxMinSizeCrownCharge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>75</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QWidget" name="buttomAndProgressBarWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonStartAnalysis">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Начать анализ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Действия</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Интервалы дефектов</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="autoScrollMargin">
           <number>10</number>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionHideAnalysisPanel">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/exitPAnel.png</normaloff>:/AnalysisPanelButton/exitPAnel.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть панель анализа</string>
   </property>
   <property name="toolTip">
    <string>Закрыть панель анализа</string>
   </property>
  </action>
  <action name="actionAddIntervalComment">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/addNewPart.png</normaloff>:/AnalysisPanelButton/addNewPart.png</iconset>
   </property>
   <property name="text">
    <string>Добавить замечание</string>
   </property>
   <property name="toolTip">
    <string>Добавить замечание</string>
   </property>
  </action>
  <action name="actionSaveVideoFragment">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/savevid.png</normaloff>:/AnalysisPanelButton/savevid.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить видео фрагмент</string>
   </property>
   <property name="toolTip">
    <string>Сохранить видео фрагмент</string>
   </property>
  </action>
  <action name="actionExcelExport">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/excel.png</normaloff>:/AnalysisPanelButton/excel.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт в Excel</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/clear.png</normaloff>:/AnalysisPanelButton/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="toolTip">
    <string>Очистить</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/delete.png</normaloff>:/AnalysisPanelButton/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </action>
  <action name="actionDeleteAndPlayNext">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/playvperedAndDel.png</normaloff>:/AnalysisPanelButton/playvperedAndDel.png</iconset>
   </property>
   <property name="text">
    <string>Удалить и воспроизвесьти следующее</string>
   </property>
  </action>
  <action name="actionPlayNext">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/playvpered.png</normaloff>:/AnalysisPanelButton/playvpered.png</iconset>
   </property>
   <property name="text">
    <string>Воспроизвести следующий</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/play.png</normaloff>:/AnalysisPanelButton/play.png</iconset>
   </property>
   <property name="text">
    <string>Воспроизвести</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/stop.png</normaloff>:/AnalysisPanelButton/stop.png</iconset>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="actionPlayPrev">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/playback.png</normaloff>:/AnalysisPanelButton/playback.png</iconset>
   </property>
   <property name="text">
    <string>Воспроизвести предыдущий фрагмент</string>
   </property>
  </action>
  <action name="actionDisplayFrame">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/AnalysisPanelButton/sm.png</normaloff>:/AnalysisPanelButton/sm.png</iconset>
   </property>
   <property name="text">
    <string>Показать кадр</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="text">
    <string>Провести анализ</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
